src/data/family.py

```diff
@@ -263,37 +268,68 @@ def calculated_parts(self) -> int | None:
 
     @property
     def numbers_str(self):
-        name: str = 'échiquiers' if self.type in [ScreenType.Boards, ScreenType.Input, ] else 'joueur·euses'
-        match (self.first, self.last, self.number, self.parts):
-            case (None, None, None, None):
-                return 'tous les échiquiers' \
-                    if self.type in [ScreenType.Boards, ScreenType.Input, ] \
-                    else 'tou·tes les joueur·euses'
-            case (first, None, None, None) if first is not None:
-                return f'{name} à partir du n°{first}'
-            case (None, last, None, None) if last is not None:
-                return f"{name} jusqu'à n°{last}"
-            case (first, last, None, None) if first is not None and last is not None:
-                return f'{name} du n°{first} au n°{last}'
-            case (None, None, number, None) if number is not None:
-                return f'écrans de {number} {name}'
-            case (first, None, number, None) if first is not None and number is not None:
-                return f'écrans de {number} {name} à partir du n°{first}'
-            case (None, last, number, None) if last is not None and number is not None:
-                return f'écrans de {number} {name} jusqu\'au n°{last}'
-            case (first, last, number, None) if first is not None and last is not None and number is not None:
-                return f'écrans de {number} {name} du n°{first} au n°{last}'
-            case (None, None, None, parts) if parts is not None:
-                return f'{name} sur {parts} écrans'
-            case (first, None, None, parts) if first is not None and parts is not None:
-                return f'{name} à partir de n°{first}, sur {parts} écrans'
-            case (None, last, None, parts) if last is not None and parts is not None:
-                return f'{name} jusqu\'au n°{last}, sur {parts} écrans'
-            case (first, last, None, parts) if first is not None and last is not None and parts is not None:
-                return f'{name} du n°{first} au n°{last}, sur {parts} écrans'
-            case _:
-                raise ValueError(
-                    f'first={self.first}, last={self.last}, parts={self.parts}, number={self.number}')
+        if self.type in [ScreenType.Boards, ScreenType.Input]:
```

Comment:
You don't need to use a list when you can use a tuple:
```suggestion
        if self.type in (ScreenType.Boards, ScreenType.Input):
```